AI/pyneogame/Engine.py

```diff
@@ -148,83 +148,17 @@ def test_player(self, agent):
         Arguments:
             agent {BaseAgent} -- Agent class inheriting from BaseAgent
         """
+        for _ in range(1000):
+            self.deal_cards()
+            agent_action = agent.get_action(self.get_player_state(),
+                                            actions=self.get_actions())
 
-        self.deal_cards()
-        agent_action = agent.get_action(self.get_player_state(),
-                                        actions=self.get_actions())
-
-        # Check that action recieved is a numpy array
-        assert isinstance(agent_action, np.ndarray)
+            # Check that action recieved is a numpy array
+            assert isinstance(agent_action, np.ndarray)
 
-        # Check that the player action produces the required number of cards
-        # Also checks that action can be convertet to boolean array
-        try:
+            # Check that the player action produces the required number of cards
+            # Also checks that action can be convertet to boolean array
+            
             assert len(self.player_hand[agent_action.astype(
```

Comment:
Add the comment as message i.e. assert statement, message

Small typo in the comment.